src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/util/TestStorageHelper.java

```diff
@@ -0,0 +1,109 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.util;
+
+import com.microsoft.azure.storage.CloudStorageAccount;
+import com.microsoft.azure.storage.StorageException;
+import com.microsoft.azure.storage.blob.CloudBlobClient;
+import com.microsoft.azure.storage.blob.CloudBlobContainer;
+import com.microsoft.azure.storage.blob.CloudBlockBlob;
+import org.testcontainers.containers.DockerComposeContainer;
+
+import java.io.File;
+import java.net.URISyntaxException;
+import java.security.InvalidKeyException;
+
+import static uk.gov.hmcts.reform.bulkscanprocessor.helper.DirectoryZipper.zipDir;
+
+public class TestStorageHelper {
+
+    private static TestStorageHelper INSTANCE;
+
+    public static final String CONTAINER_NAME = "bulkscan";
+    public static final String ZIP_FILE_NAME = "1_24-06-2018-00-00-00.zip";
+
+    private static DockerComposeContainer<?> dockerContainer;
+    public static CloudBlobClient cloudBlobClient;
+    private CloudBlobContainer testContainer;
+
+    private TestStorageHelper() {
+        // empty constructor
+    }
+
+    public static TestStorageHelper getInstance() {
+        if (INSTANCE == null) {
+            INSTANCE = new TestStorageHelper();
+        }
+
+        return INSTANCE;
+    }
+
+    public static void createDocker() {
+        dockerContainer = new DockerComposeContainer<>(
```

Comment:
why not use Class rule
```
@ClassRule
public static DockerComposeContainer environment =
    new DockerComposeContainer(new File("src/test/resources/compose-test.yml"))
            .withExposedService("redis_1", REDIS_PORT)
            .withExposedService("elasticsearch_1", ELASTICSEARCH_PORT);

```

Reply:
`ClassRule` is junit4 and we should finally make progress to junit5 instead ;). it's tricky as it needs a lot of other reviews to be done beforehand ;)